### PR TITLE
[LC-717] add icon_rc version in global configuration

### PR DESCRIPTION
**Pull request by laurenceyoon**
`icon_rc` version is also added in ICON_VERSIONS config.

### CODE REVIEW

**Review by metalg0su — APPROVED** [review]

File: loopchain/configure.py [hunk 1]
```diff
@@ -148,6 +148,10 @@ def _set_package_version(self):
         for package in filtered_pkg:
             version = pkg_resources.get_distribution(package).version
             globals()['ICON_VERSIONS'][package] = version
+        command_result = os.popen('icon_rc -version').read()  # 'icon_rc vX.X.X, tags()-YYYY-MM-DD-00:00:00\n'
+        if command_result:
+            icon_rc_version = command_result.split(' ')[1].split(',')[0]
```
Comment (yakkle) [inline loopchain/configure.py]: why don't you using `regex`?

Comment (laurenceyoon) [reply loopchain/configure.py]: I've changed it to regex. Thanks!

Comment (laurenceyoon) [reply loopchain/configure.py]: I've changed it to regex. Thanks

File: loopchain/configure.py [hunk 2]
```diff
@@ -148,6 +148,11 @@ def _set_package_version(self):
         for package in filtered_pkg:
             version = pkg_resources.get_distribution(package).version
             globals()['ICON_VERSIONS'][package] = version
+        command_result = os.popen('icon_rc -version').read()
+        match_result = re.match(r'([a-z_]+)\s([\da-zA-Z-_\.]+)', command_result)  # ('icon_rc', 'vX.X.X')
+        if match_result:
+            icon_rc_version = match_result.groups()[1]
```
Comment (yakkle) [inline loopchain/configure.py]:
```suggestion
            icon_rc_version = match_result.group(2)
```

**Review by yakkle — APPROVED** [review]
It would be better squash before merge.

**Pull request merged**